docs/wiki/references/Glosarry.md

```diff
@@ -0,0 +1,60 @@
+> \[!WARNING\]
+> This page is a work in progress.
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Terms](#terms)
+  - [Event streaming](#event-streaming)
```

Comment:
remove:
- event streaming
- real-time streaming data
- basket


add:
- realtime
- "wiring" or "graph building" time
- graph run time
- ticked (as in `csp.ticked`)
- valid (as in `csp.valid`)
- push mode
- edge
- delayed edge
- feedback
- struct
- list basket
- dict basket
- dynamic graph
- push input adapter
- pull input adapter
- output adapter
- managed sim adapter
- adapter manager

Reply:
I think a glossary is an excellent idea and sorely needed!

Reply:
Thank you! Added the terms as TODOs to the page. Kept "event streaming" alone from your to-remove list because I think it's useful to create a source of truth for vocabulary used throughout the docs. We can remove it if you feel strongly about it. :)